### DISCUSSION TITLE
Adding own experiment in Inputparamter and datafitting notebook

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3294] [Q&A]
In place of solving the model, I need to simulate the model with a custom-defined experiment in the notebook. What changes would be required in the notebook?

![Screenshot 2023-08-24 051600](https://github.com/pybamm-team/PyBaMM/assets/95225799/f3c85ddc-9914-49f8-8bf0-a1bcdfd0ea56)

**Comment by brosaplanella** [comment]
To add an experiment please see [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-5-run-experiments.html). Note that this uses the simulation class, which is more compact, but you can pass the solver and parameter values to the simulation (see the [simulation docs](https://docs.pybamm.org/en/latest/source/api/simulation.html#simulation)).
